grpc/oob/bmc/bmc.go

```diff
@@ -381,6 +385,9 @@ func (m Action) DeactivateSOL(ctx context.Context) error {
 	span.SetAttributes(attribute.String("bmc.host", host), attribute.String("bmc.username", user))
 	m.SendStatusMessage("working on SOL session deactivation")
 
+	ctx, cancel := context.WithTimeout(ctx, 120*time.Second)
+	defer cancel()
+
 	opts := []bmclib.Option{
 		bmclib.WithLogger(m.Log),
 		bmclib.WithPerProviderTimeout(common.BMCTimeoutFromCtx(ctx)),
```

Comment:
To confirm - there is currently an option being passed in which sets a timeout for each of the providers, is this not working?
 `bmclib.WithPerProviderTimeout(common.BMCTimeoutFromCtx(ctx))`
 
With the current change - if a provider takes too long on the connection it leaves no time for the others.